tests/test_cookie_storage_with_bson.py

```diff
@@ -0,0 +1,179 @@
+import asyncio
+import bson.json_util as json
+from bson import objectid
+import socket
+import unittest
+import time
+
+from aiohttp import web, request
+from aiohttp_session import (Session, session_middleware,
+                             get_session, SimpleCookieStorage)
+
+
+class TestSimleCookieStorage(unittest.TestCase):

```

Comment:
Also no need to copy paste those files, just add simple test for each storage type that hits changed logic.